### PR TITLE
Geomapper docstring and error cleanup

### PR DESCRIPTION
**Pull request by alexcoda**
### Description
Update out-of-date docstrings, some formatting, and error handling. Contains no changes to functionality.

### Changelog
- Update inaccurate docstring parameters.
- Simplify logic for error handling in `get_geos_within` method.
- Minor formatting changes

### CODE REVIEW

File: _delphi_utils_python/delphi_utils/geomap.py
Comment (krivard) [inline _delphi_utils_python/delphi_utils/geomap.py:603]:
```suggestion
        Given container_geocode (e.g "ca" for California) of type container_geocode_type
        (e.g "state"), return:
            - all (contained_geocode_type)s within container_geocode
```

File: _delphi_utils_python/delphi_utils/geomap.py
Comment (krivard) [inline _delphi_utils_python/delphi_utils/geomap.py:615]:
```suggestion
            The subregion type to get all instances of. One of "state", "county"
```

Comment (krivard) [reply _delphi_utils_python/delphi_utils/geomap.py:615]: or

```suggestion
            The subregion type to retrieve. One of "state", "county"
```

File: _delphi_utils_python/delphi_utils/geomap.py
Comment (krivard) [inline _delphi_utils_python/delphi_utils/geomap.py:636]: Include `f"({contained_geocode_type}, {container_geocode_type})"` in this message to give future encounters a leg up

File: _delphi_utils_python/delphi_utils/geomap.py
Comment (krivard) [inline _delphi_utils_python/delphi_utils/geomap.py:628]: so i'm generally in favor of putting quick test+exit/raise behavior before slow+process behavior to reduce cognitive load while reading, which would mean rejecting this particular change. in this case i see the benefit of a single exit condition based on more complex logic, but just to keep in mind for future prs

**Review by krivard — CHANGES_REQUESTED** [review]
Greetings! I'm being picky here because it's your first PR, but please know that these kinds of changes are _extremely welcome_ and bring incredible joy to my inner tech writer/codesmith 🙌

File: _delphi_utils_python/delphi_utils/geomap.py
Comment (alexcoda) [reply _delphi_utils_python/delphi_utils/geomap.py]: Added this into the message!

**Review by krivard — APPROVED** [review]
👍  nice work!

**Comment by krivard** [comment]
You're failing linting, though it looks like that's been fixed in #1232.

Lawful-good option: Cherry-pick 28ed0299 into this branch and drop it from #1232 
Chaotic-good option: I wait for Dmitry's okay on #1232, then just merge both PRs in quick succession

No strong preference here, up to you

**Comment by chinandrew** [comment]
> Chaotic-good option: I wait for Dmitry's okay on #1232, then just merge both PRs in quick succession
> No strong preference here, up to you

#1232 contains all the commits in this PR due to an inability to merge from a forked branch into another forked branch on this repo, so once that one gets approved we could just merge it and close this one.

**Pull request merged**
